marklogic-data-hub/src/test/resources/test-projects/custom-step-with-sleep/modules/root/custom-modules/custom/customStepWithSleep/customStepWithSleep.sjs

```diff
@@ -0,0 +1,8 @@
+function main(contentItem, options) {
+  xdmp.sleep(100000);
```

Comment:
Sorry Rob.  That was me saving my current work because I was still trying to work out why Eclipse doesn't like my project.  I just updated to my v1 of the test.  Reopening the PR